static/js/signal.js

```diff
@@ -0,0 +1,32 @@
+window.onload = function() {
+  updateSignalLoop()
+};
+
+async function updateSignalLoop() {
+  const refreshTime =2 * 1000 
+
+  var xmlHttp = new XMLHttpRequest();
+  xmlHttp.open("GET", "/api/signal-strength", true);
+  xmlHttp.setRequestHeader("Authorization", "Basic "+btoa("admin:feathers"))
+  xmlHttp.onload = async function() {
+    if (xmlHttp.status == 200) {
+      var signalElement = document.getElementById("signal-strength");
+      signalElement.innerText = xmlHttp.response;
+      await sleep(refreshTime);
+    } else {
+      console.log("status:", xmlHttp.status);
+      console.log("response:", xmlHttp.response);
```

Comment:
probably don't want to log this continuously

Reply:
Changed to only log first error at start or after being successfull